src/Symfony/Component/Ldap/Entry.php

```diff
@@ -22,9 +22,24 @@ class Entry
     public function __construct(string $dn, array $attributes = array())
     {
         $this->dn = $dn;
+        $attributes = $this->resetValuesIndex($attributes);
         $this->attributes = $attributes;
     }
 
+    /**
+     * Resets the index for array values.
+     *
+     * @param array $attributes
+     *
+     * @return array
+     */
+    private function resetValuesIndex(array $attributes = array())
```

Comment:
No need for a default value there. The method is only called internally, and there's never any case where it will be called without arguments (the initial value of the attributes is only set in the constructor).

Reply:
done.